pkg/minishift/openshift/version/openshift_versions.go

```diff
@@ -91,19 +92,23 @@ func PrintDownStreamVersions(output io.Writer, minSupportedVersion string) {
 }
 
 func PrintUpStreamVersions(output io.Writer, minSupportedVersion string) {
-	resp, err := getResponseBody("https://registry.hub.docker.com/v2/repositories/openshift/origin/tags/")
+	dockerRegistryUrl := "https://registry.hub.docker.com/v2/repositories/openshift/origin/tags/?page_size=1024"
+	resp, err := getResponseBody(dockerRegistryUrl)
 	if err != nil {
 		fmt.Fprintf(output, "Error Occured %s", err)
 		return
 	}
 	defer resp.Body.Close()
 	decoder := json.NewDecoder(resp.Body)
 	var data ImageTags
-	err = decoder.Decode(&data)
-	if err != nil {
-		fmt.Printf("%T\n%s\n%#v\n", err, err, err)
-		return
-	}
```

Comment:
The New URL JSON has `last_updated` field empty, hence it was returning following error. I am not sure how should we handle the error as we do not care about the `last_updated` field. 
```
*time.ParseError
parsing time "null" as ""2006-01-02T15:04:05Z07:00"": cannot parse "null" as """
&time.ParseError{Layout:"\"2006-01-02T15:04:05Z07:00\"", Value:"null", LayoutElem:"\"", ValueElem:"null", Message:""}
```